docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md

```diff
@@ -8,108 +8,138 @@ updatedAt: "2022-12-13T20:17:44.370Z"
 
-- Node builder - version 6.x
-- React builder - version 3.x
-- Admin builder - version 1.x
-- Messages builder - version 1.x
-- Docs builder- version 0.x
-- Graphql builder -version 1.x
+## Data privacy
 
-### Front-End
+### Persisting PII within VTEX
 
-To ensure your front-end code is performant, take the following recommendations:
+If using Master Data or VBase to store Personal Identifiable Information (PII), a mechanism must be in place to ensure compliance with the [Right to be Forgotten](https://www.gdpreu.org/right-to-be-forgotten/) or other similar data-protection practices. Possible solutions are to provide an option or endpoint in the app that erases the data, for instance, by using a delete operation with our Clients ([Master Data](https://developers.vtex.com/docs/guides/create-master-data-crud-app#delete), [VBase](https://github.com/vtex/node-vtex-api/blob/1b3c54976aa974619d0728fae4ed2fe076dbb551/src/clients/infra/VBase.ts#L178)) or through the API ([Master Data v1](https://developers.vtex.com/docs/api-reference/masterdata-api#delete-/api/dataentities/-acronym-/documents/-id-), [Master Data v2](https://developers.vtex.com/docs/api-reference/master-data-api-v2#delete-/api/dataentities/-dataEntityName-/documents/-id-)).
 
-- Use [Tachyons](https://tachyons.io/), a modern framework for writing CSS styles.
+### Sending PII to external service
 
-- Avoid external Javascript libraries when rendering component content to avoid increasing time-response for users. Give preference to the [React Hooks](https://pt-br.reactjs.org/docs/hooks-reference.html) API.
+We should ensure that all PII being shared externally is strictly necessary for the app to work. If you have any questions, reach out to us [via ticket](https://help.vtex.com/en/tutorial/opening-tickets-to-vtex-support--16yOEqpO32UQYygSmMSSAM).
 
-- Do not use hardcoded labels in your code. If your app works with text components, create a structure of constants in a `constants.ts` file and import their content dynamically. To translate these labels into other languages, always use _messages builder._
+## Usability
 
-### Security and authentication
+### Implementing internationalization (i18n) correctly
 
-At the authentication level, take the following requirements
+Ensure that all messages are [internationalized](https://developers.vtex.com/docs/guides/vtex-io-multi-language-stores#storefront-content-internationalization) using the `messages` builder. Also, i18n keys should start with the domain (`store/` for storefront and `admin/` for Admin) where they are used, otherwise they will not render properly. For more information, see our [reference](https://developers.vtex.com/docs/guides/vtex-io-documentation-8-translating-the-component) for translating components.
```

Comment:
```suggestion
Ensure that all messages are [internationalized](https://developers.vtex.com/docs/guides/vtex-io-multi-language-stores#storefront-content-internationalization) using the `messages` builder. Also, i18n keys should start with the domain (`store/` for storefront and `admin/` for Admin) where they are used. Otherwise, they will not render properly. For more information, see our [reference](https://developers.vtex.com/docs/guides/vtex-io-documentation-8-translating-the-component) for translating components.
```